packages/components/file-uploader/src/file-uploader.css

```diff
@@ -0,0 +1,421 @@
+/* General/default state................................................ */
+
+:host > .file-upload-wrapper > .file-upload-button,
+:host > .file-card-wrapper > .file-card {
+  align-items: center;
+  background-color: var(--ts-color-white);
+  border-radius: 3px;
+  border: var(--ts-border);
+  cursor: pointer;
+  display: flex;
+  height: calc(3.5 * var(--ts-unit-half));
+  justify-content: space-between;
+  max-width: 300px;
+  margin-bottom: var(--ts-unit-half);
+  overflow: hidden;
+  padding: calc(0.5 * var(--ts-unit-half)) calc(1px + var(--ts-unit-half));
+  position: relative;
+  transition: border 0.15s ease;
+  width: 300px;
+}
+
+:host > .file-upload-wrapper > .file-upload-button:hover {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)